lrpar/src/lib/ctbuilder.rs

```diff
@@ -78,21 +78,40 @@ pub enum ActionKind {
     GenericParseTree
 }
 
-struct CTConflictsError<StorageT: Eq + Hash> {
+
+pub struct ConflictsStruct<StorageT: Eq + Hash> {
     pub grm: YaccGrammar<StorageT>,
     pub sgraph: StateGraph<StorageT>,
     pub stable: StateTable<StorageT>
 }
 
+impl<StorageT> ConflictsStruct<StorageT>
```

Comment:
In retrospect, this is a weird name. Shouldn't it just be `Conflicts`?

Reply:
Yes this was kind of a placeholder, I forgot to ask what we should name this. `Conflicts` is already the name of the `Conflicts` struct within `StateTable`. Which is why I wasn't sure how to name this.

Reply:
That's another file and we don't import that struct in here I think, so reusing the name isn't a total disaster.

Reply:
Won't that be confusing from a documentation perspective. We'd have a struct `Conflicts`, which has a `StateTable`, which has a `Conflicts`. But the two `Conflicts` are not the same.

Reply:
Maybe I name the one in StateTable `GrammarConflicts`

Reply:
They'll never appear on the same page...

I'm not saying it's ideal, but we can't add `Struct` as a suffix and I can't think of a better name...

Reply:
See my above suggestion to rename the `StateTable` one to `GrammarConflicts`.

Reply:
They're all grammar conflicts in one way...

If you really want to separate the names, maybe make `StateTable`'s struct be `STConflicts`?

Reply:
Alternatively we can name the CTParserBuilder one `CTConflicts`?